src/Arcus.Templates.AzureFunctions.Databricks.JobMetrics/Startup.cs

```diff
@@ -39,7 +39,7 @@ public override void Configure(IFunctionsHostBuilder builder)
                 stores.AddAzureKeyVaultWithManagedIdentity("https://your-keyvault.vault.azure.net/", CacheConfiguration.Default);
             });
 
-            var instrumentationKey = config.GetValue<string>("APPLICATIONINSIGHTS_INSTRUMENTATIONKEY");
+            var instrumentationKey = config.GetValue<string>("APPINSIGHTS_INSTRUMENTATIONKEY");
```

Comment:
Just wondering if we have settings file that need to be changed as well ?

Reply:
In this case, no, bc this is set via the integration test project, and this was already done in this PR.
So, it's like this:

Azure DevOps library >-sets-> Integration Test project >-sets-> Project template.